### PR TITLE
Update devops

### PR DESCRIPTION
**Pull request by thanmayi565**

### CODE REVIEW

**Review by ashachowdary2001 — APPROVED** [review]

**Review by BandiBabuReddy — APPROVED** [review]

**Comment by AWS-GitHub-Practice** [comment]
conflict resolved

**Comment by AWS-GitHub-Practice** [comment]
conflict resolved

File: devops [hunk 1]
```diff
@@ -7,5 +7,5 @@ This is second change from sushma
 salaar relaesing tomarrow
```
Comment (AWS-GitHub-Practice) [inline devops:9]: Salaar released today

**Review by AWS-GitHub-Practice — COMMENTED** [review]
Salar is released

**Pull request merged**